### PR TITLE
Languages

### PR DESCRIPTION
**Pull request by geoand**
Added a section on alternative JVM languages

### CODE REVIEW

**Comment by pacher** [comment]
@geoand Why didn't you include Xtend (http://xtend-lang.org/)?

**Comment by geoand** [comment]
Honest mistake :-)

File: README.md [hunk 1]
```diff
@@ -59,6 +60,16 @@ A curated list of awesome Java frameworks, libraries and software.
     - [Websites](#websites)
 - [Contributing](#contributing)
 
+## Alternative Programming Languages
```
Comment (akullpp) [inline README.md]: I'm against the title, "Languages" would be sufficient and it wouldn't be placed as the first item...

File: README.md [hunk 2]
```diff
@@ -59,6 +60,16 @@ A curated list of awesome Java frameworks, libraries and software.
...
+
+*Languages other than Java that can be used to write JVM applications*
+
+* [Scala](http://www.scala-lang.org/) - Statically typed programming language that fuses the object - oriented model and functional programming ideas.
+* [Groovy](http://www.groovy-lang.org/) - Optionally typed and dynamic language, with static-typing and static compilation capabilities.
+* [Clojure](http://clojure.org/) - Dynamically typed programming language that can be seen as a modern take on Lisp.
+* [Ceylon](http://ceylon-lang.org/) - Statically typed object - oriented language developed by RedHat.
```
Comment (akullpp) [inline README.md]: Remove the whitespaces left and right, so that it reads `object-oriented`

File: README.md [hunk 3]
```diff
@@ -59,6 +60,16 @@ A curated list of awesome Java frameworks, libraries and software.
     - [Websites](#websites)
 - [Contributing](#contributing)
 
+## Alternative Programming Languages
+
+*Languages other than Java that can be used to write JVM applications*
+
+* [Scala](http://www.scala-lang.org/) - Statically typed programming language that fuses the object - oriented model and functional programming ideas.
```
Comment (akullpp) [inline README.md]: Same here `object-oriented`

File: README.md [hunk 4]
```diff
@@ -59,6 +60,16 @@ A curated list of awesome Java frameworks, libraries and software.
...
+* [Groovy](http://www.groovy-lang.org/) - Optionally typed and dynamic language, with static-typing and static compilation capabilities.
+* [Clojure](http://clojure.org/) - Dynamically typed programming language that can be seen as a modern take on Lisp.
+* [Ceylon](http://ceylon-lang.org/) - Statically typed object - oriented language developed by RedHat.
+* [Kotlin](http://kotlinlang.org/) - Statically typed programming language for the JVM, Android and the browser.
```
Comment (akullpp) [inline README.md]: A bit more information would be nice, not just typing.

Comment (geoand) [inline README.md]: You're right. I simply added a small description found in the language's
website. I'll fix at the first possible moment!

**Comment by geoand** [comment]
Updated

**Pull request merged**